### PR TITLE
Application curriculum

### LINKED ISSUE #492: Replace external requirements with our own
Khan Academy offers us the opportunity to assess the progress of learners in a more fine-grained way than platforms like, say, freeCodeCamp. Their [Computing Curriculum](https://www.khanacademy.org/computing) looks good and is based around JavaScript.

There is a case for starting our learners off with [Computer programming](https://www.khanacademy.org/computing/computer-programming) (up to _HTML/JS: Making webpages interactive_) as well as [Computers and the Internet](https://www.khanacademy.org/computing/computers-and-internet). 

We are currently trialling [Algorithms](https://www.khanacademy.org/computing/computer-science/algorithms) with FAC22 (see #485). They are finding it challenging, but in terms of difficulty, it is probably at a level suitable for preparing for entry to the job market. They have also been set optional homework of [Get Ready for Algebra 1](https://www.khanacademy.org/math/get-ready-for-algebra-i), as a remedial intro to algebra.

### PR DESCRIPTION
**Pull request by Albadylic**
Fixes #492 

- All weekly prompts on the coursebook
- Information on how to complete them, what platforms to use and how to approach problems
- Added to navigation on left of syllabus

### CODE REVIEW

File: src/course/application/git.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# git and GitHub
```
Comment (oliverjam) [inline src/course/application/git.md:1]:
```suggestion
# Git and GitHub
```
The official Git website capitalises it so we probs should too

File: src/course/application/platforms.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+[Stackblitz](https://stackblitz.com/) is an IDE which has a similar look and feel to VS Code. From here you can build websites and synchronise your code with a GitHub repository.
+
+Stackblitz doesn't save your work automatically so remember to click Save in the top left after you write a chunk of code.
+
+## VS Code
+
+[VS Code](https://code.visualstudio.com/) is a code editor widely used in Software Development. Using VS Code allows you to save your code to your local machine.
+
+VS Code has numerous advantages such as source control, extentions, and keyboard shortcuts.
```
Comment (oliverjam) [inline src/course/application/platforms.md:19]:
```suggestion
VS Code has numerous advantages such as source control, extensions, and keyboard shortcuts.
```

File: src/course/application/feature.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+## Build a feature
```
Comment (oliverjam) [inline src/course/application/feature.md:1]:
```suggestion
# Build a feature
```

File: src/course/application/flexbox.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+## Flexbox
```
Comment (oliverjam) [inline src/course/application/flexbox.md:1]:
```suggestion
# Flexbox
```

**Review by oliverjam — CHANGES_REQUESTED** [review]
Couple of small typos fixes

**Comment by yvonne-liu** [comment]
✨

**Pull request merged**